README.md

```diff
@@ -553,6 +553,32 @@ And attach any image with the following
 The width will be the commented html after the image (in this case 300px).
 
 
+### Usage with the mermaid-cloud plugin
+
+> Note: The mermaid-cloud plugin needs to be added to your Confluence instance
+
+This plugin uses attachments to provide and place the graph in your confluence page, so don't forget to add the attachment.
+
+```mardown
+<!-- Attachment: graph.mermaid -->
+<!-- Macro: :mermaid:(.+):(.*):(.*)
```

Comment:
Before we start this type of workarounds with custom macros, I think it would be cleaner to have a --mermaid-provider CLI flag  and render fenced codeblocks with `mermaid` language accordingly.

Reply:
or maybe even specify MERMAID_PROVIDER via environment variables so it's easier to adjust these settings for CI/CD users

Reply:
Sounds good! I want to work towards a PR that includes https://github.com/dreampuf/mermaid.go / https://github.com/kovetskiy/mark/pull/167 and will add support for multiple implementations there.

Reply:
Sounds like a plan, am going to close this PR and open a new one once everything is ready.